src/vtbackend/Terminal.cpp

```diff
@@ -163,6 +165,8 @@ Terminal::Terminal(Events& eventListener,
     setMode(DECMode::TextReflow, true);
     setMode(DECMode::SixelCursorNextToGraphic, true);
 #endif
+    _supportedVTSequences.enableSequence(SCOSC);
+    _supportedVTSequences.disableSequence(DECSLRM);
```

Comment:
```suggestion
    setMode(DECMode::LeftRightMargin, false);
```

This way we could avoid code duplication of actually important logic, reducing potential bugs in the future. :)